hack/check-file-length.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -ex
+
+filename=$1
+if [ -z "$filename" ]; then
+    echo >&2 $0 expects a filename
+    exit 1
+fi
+
+token=lint_maximum_file_length
+max=$(grep "$token" "$filename" | awk '{print $3}')
+len=$(wc -l < "$filename")
+
+if [ -z "$max" ]; then
+    echo >&2 $filename: $token not found
+    exit 1
+fi
+
+if [ "$len" -gt "$max" ]; then \
```

Comment:
I think the \ could be removed, right? It looks odd that it's not used in the previous condition.

Reply:
also on line 23